pom.xml

```diff
@@ -44,41 +57,9 @@
             <artifactId>commons-digester3</artifactId>
             <version>3.2</version>
         </dependency>
-        <!--Build breaks without these:-->
         <dependency>
-            <groupId>jfree</groupId>
-            <artifactId>jcommon</artifactId>
-            <version>1.0.16</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>jfree</groupId>
-            <artifactId>jfreechart</artifactId>
-            <version>1.0.13</version>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
Just to clarify: you wrote that plugin pom is for dependencies in the `test` scope, but these are in the `provided` scope. Can you confirm they're no longer needed?

Reply:
> Just to clarify: you wrote that plugin pom is for dependencies in the `test` scope, but these are in the `provided` scope. Can you confirm they're no longer needed?

I thought that passing compilation and passing automated tests were enough to indicate that those dependencies no longer need to be declared in the pom file.

When I run `mvn help:effective-pom`, I see the following output:

```
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.jfree</groupId>
        <artifactId>jfreechart</artifactId>
        <version>1.0.19</version>
      </dependency>
      <dependency>
        <groupId>org.jfree</groupId>
        <artifactId>jfreechart</artifactId>
        <version>1.0.19</version>
      </dependency>
    </dependencies>
  </dependencyManagement>
```

I think that indicates that the parent pom is providing that dependency now.  The plugin does not need to declare it separately because it has it from the parent pom.  I did remove many other lines from the effective pom output, but that was for clarity in reading.